api/v2010/short_code/read-example-2/read-example-2.3.x.js

```diff
@@ -5,7 +5,7 @@ const authToken = 'your_auth_token';
 const client = require('twilio')(accountSid, authToken);
 
 client.shortCodes.each({
-                    shortCode: '67898'
+                    shortCode: 67898
```

Comment:
This seems wrong @joliveros -- short_code is defined as a string in the api definitions